### PR TITLE
Add list of canary tests to openshift-tests image

### PR DESCRIPTION
**Pull request by bparees**
Adds a list of tests that pass with extremely high reliability.  We will
run these tests in a new e2e job.  The origina list was generated
by taking tests that have passed at a rate of 99% or higher over the last week,
from sippy data.

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton @stevekuznetsov ptal at this and https://github.com/openshift/release/pull/16595 and tell me if this is an acceptable way to do this.  I'm open to suggestions but fundamentally i want to maintain a text file list of tests somewhere that is release-branch managed, and then have a scheduled CI job (and maybe in the future a PR-based CI job also) that runs those tests.

**Comment by deads2k** [comment]
how will you diff this list over time as tests move in and out?

File: cmd/openshift-tests/e2e.go [hunk 1]
```diff
@@ -340,6 +340,34 @@ var staticSuites = testSuites{
 		},
 		PreSuite: suiteWithProviderPreSuite,
 	},
+	{
+		TestSuite: ginkgo.TestSuite{
+			Name: "experimental/reliability/minimal/parallel",
```
Comment (smarterclayton) [inline cmd/openshift-tests/e2e.go:345]: Why not combine these into a single run (serial + parallel will automaticalyl get run correctly)?

Comment (bparees) [reply cmd/openshift-tests/e2e.go:345]: i didn't realize it would do that automatically, sure i can combine.

Comment (bparees) [reply cmd/openshift-tests/e2e.go:345]: (but why do we have separate serial jobs today, then?)

Comment (bparees) [reply cmd/openshift-tests/e2e.go]: i mean i based this on:

```
	{
		TestSuite: ginkgo.TestSuite{
			Name: "openshift/conformance/parallel",
			Description: templates.LongDesc(`
		Only the portion of the openshift/conformance test suite that run in parallel.
		`),
			Matches: func(name string) bool {
				if isDisabled(name) {
					return false
				}
				return strings.Contains(name, "[Suite:openshift/conformance/parallel")
			},
			Parallelism:          30,
			MaximumAllowedFlakes: 15,
			SyntheticEventTests:  ginkgo.JUnitForEventsFunc(synthetictests.StableSystemEventInvariants),
		},
		PreSuite: suiteWithProviderPreSuite,
	},
	{
		TestSuite: ginkgo.TestSuite{
			Name: "openshift/conformance/serial",
			Description: templates.LongDesc(`
		Only the portion of the openshift/conformance test suite that run serially.
		`),
			Matches: func(name string) bool {
				if isDisabled(name) {
					return false
				}
				return strings.Contains(name, "[Suite:openshift/conformance/serial") || isStandardEarlyOrLateTest(name)
			},
			SyntheticEventTests: ginkgo.JUnitForEventsFunc(synthetictests.StableSystemEventInvariants),
		},
		PreSuite: suiteWithProviderPreSuite,
	},
```

**Comment by bparees** [comment]
updated to a single suite after offline discussion with @smarterclayton, self-tagging.

**Pull request merged**